server/router/router.go

```diff
@@ -187,8 +187,6 @@ func Load(serveHTTP func(w http.ResponseWriter, r *http.Request), middleware ...
 		debugger.GET("/pprof/trace", debug.TraceHandler())
 	}
 
-	e.GET("/api/web-config.js", web.WebConfig)
```

Comment:
@6543 This was needed for the new UI and works as a replacement for the old template injection.

Reply:
@anbraten sorry for that one